modules/navier_stokes/test/tests/postprocessors/tests

```diff
@@ -0,0 +1,86 @@
+[Tests]
+  design = 'VolumetricFlowRate.md InternalVolumetricFlowRate.md'
+  issues = '#16169 #16585'
+  [fe]
+    type = CSVDiff
+    input = conservation_INSFE.i
+    csvdiff = straight_INSFE.csv
+    abs_zero = 1e-8
+    requirement = 'The system shall be able to compute mass and momentum flow rates at internal and external boundaries of a straight channel with a finite element incompressible Navier Stokes model.'
```

Comment:
I am curious whether you observe conservation for FE?

Reply:
Nah not at all. Like it's abysmal with such a coarse mesh too